docs/sources/get-started/ride-share-tutorial.md

```diff
@@ -0,0 +1,274 @@
+     basic_auth_password = basic_auth_password, # for grafana cloud
+     tags             = {
+         "region":   f'{os.getenv("REGION")}',
+     }
+ )
+```
+This tag is considered static is because the tag is set at the start of the application and doesn't change.
+In this case, it's useful for grouping profiles on a per region basis, which lets yousee the performance of the application per region.
+
+Open Grafana to see how this tag is used:
+
+1. Open the Profile Explorer in Grafana UI, which can be accessed using the following url: [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
+2. Select on `Labels` in the `Exploration` path.
+3. Select the `region` tab in the `Group by labels` section. 
+
+You should now see a list of regions that the application is running in. You can see that `eu-north` is experiencing the most load. 
+
+{{< figure max-width="100%" src="/media/docs/pyroscope/ride-share-tag-region-2.png" caption="Region Tag" alt="Region Tag" >}}
+
+Next, look at a dynamic tag use case. Within the `utils.py` file,  find the following function:
+
+```python
+ def find_nearest_vehicle(n, vehicle):
+     with pyroscope.tag_wrapper({ "vehicle": vehicle}):
+         i = 0
+         start_time = time.time()
+         while time.time() - start_time < n:
+             i += 1
+         if vehicle == "car":
+             check_driver_availability(n)
+```
+
+This example uses `tag_wrapper` to tag the function with the vehicle type.
+Notice that the tag is dynamic as it changes based on the vehicle type.
+This is useful for grouping profiles on a per vehicle basis. Allowing us to see the performance of the application per vehicle type being requested.
+
+Use the Pyroscope UI to see how this tag is used:
+1. Open the Profile Explorer in Grafana UI, which can be accessed using the following url: [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
+1. Select on `Labels` in the `Exploration` path.
+1. Select the `vehicle` tab in the `Group by labels` section. 
+
+You should now see a list of vehicle types that the application is using. You can see that `car` is experiencing the most load. 
+
+<!-- INTERACTIVE page step2.md END -->
+
+<!-- INTERACTIVE page step3.md START -->
+
+## Identifying the performance bottleneck
+
+The first step when analyzing a profile outputted from your application, is to take note of the largest node which is where your application is spending the most resources. To discover this, you can use the `Flame Graph` view:
+
+1. Open the Profile Explorer in Grafana UI, which can be accessed using the following url: [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
+1. Select the `Single View` tab.
+1. Make sure `flask-ride-sharing-app:process_cpu:cpu` is selected in the dropdown menu.
```

Comment:
Because we're expecting people to walk through this get started guide, do we need to say to return to the Grafana UI again? Maybe we just need to say In Explore Profiles... or Return to Explore Profiles?

Also, these steps referred to Single view, from the old Pyroscope UI. I've updated it to Explore Profiles. Does this do what you want for these steps?

```suggestion
1. Open Explore Profiles using the following url: [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
1. Select **Profile types** from the **Exploration** path.
1. Make sure `flask-ride-sharing-app:process_cpu:cpu` is selected in the **Service** dropdown menu.
```